scripts/generate-tts-languages

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.."
+
+python3 -m pip install --upgrade hass_nabucasa "josepy<2.0"
```

Comment:
josepy should not needed, that is a bug which is fixed here <https://github.com/NabuCasa/hass-nabucasa/pull/850>

Reply:
```suggestion
python3 -m pip install --upgrade hass_nabucasa
```